README.md

```diff
@@ -78,7 +78,7 @@ In addition to bugs.python.org, we are also hosting two other instances:
 * Jython is already using [GitHub issues](https://github.com/jython/jython/issues)
 * Still has a legacy instance of Roundup at https://bugs.jython.org/ (bjo)
 * [Thread on the Jython ML that suggests to switch to GitHub issues but keep bugs.jython.org around](https://sourceforge.net/p/jython/mailman/jython-dev/thread/CAOhO%3DaNp9oPkMwO58vK0L5gqL_nG8UD0mWom%3D1XLqxdbyfU34w%40mail.gmail.com/#msg37020624)
-* They are also using bjo for the CLA
+* They are using [the form on python.org](https://www.python.org/psf/contrib/contrib-form) to sign the CLA and bpo to check it.
```

Comment:
`bpo` or `bjo`?
Do Jython users have to create a `bpo`/`bjo` account in order to contribute?

Reply:
b*p*o is what I mean. I almost made this a one-character PR.

I'm pretty sure bpo is not open to new accounts, but we used to need such an account. We're using GitHub issues now. A b*j*o account is not necessary. (You are right that history is useful, read-only except maybe to cross-reference GitHub.)

It follows that we can ask people to sign the CLA, and take their word for that, but cannot check the CLA status of anyone new (noticed in April) or easily remind ourselves. Thanks to @Mariatta we still have a way to check the names from before April.

Reply:
> I'm pretty sure bpo is not open to new accounts, but we used to need such an account -- just to register the CLA. 

That's correct -- it is no longer possible to create new accounts on bpo.  I wasn't aware that you were relying on it, nor that you still are even after your migration.

> It follows that we can ask people to sign the CLA, and take their word for that, but cannot check the CLA status of anyone new

This sounds problematic, especially since it might have legal implications.  Have you considered switching to the [`cla-bot`](https://github.com/ambv/cla-bot) we are using, or the upstream version? (cc @ambv)

> Thanks to @Mariatta we still have a way to check the names from before April.

Just out of curiosity -- how are you checking it?

Reply:
> This sounds problematic, especially since it might have legal implications.

Yes it is. Fixing the bit of text that claims it isn't seemed a sensible move.

> Have you considered switching to the [cla-bot](https://github.com/ambv/cla-bot) we are using, or the upstream version?

I'd love us to do this. It was not possible when I first asked. Making it so seemed to be a possibility but my follow-up enquiries must be going into spam.

> Just out of curiosity -- how are you checking it?

A very minimalist Rest API on the bpo database (or a copy?). Not sure I should publish the link.

Reply:
> Yes it is. Fixing the bit of text that claims it isn't seemed a sensible move.

Sure, I will merge this PR then.

> I'd love us to do this. It was not possible when I first asked. Making it so seemed to be a possibility but my follow-up enquiries must be going into spam.

Since @ambv replied, I hope you will be able to switch to the cla-bot soon!

> A very minimalist Rest API on the bpo database (or a copy?). Not sure I should publish the link.

That's probably the same API we used to use.